basics/src/main/kotlin/de/thermondo/basics/Basics5.kt

```diff
@@ -0,0 +1,9 @@
+package de.thermondo.basics
+
+/**
+ * Create a class named Basics5
```

Comment:
Before we define the task to be done, we need to explain how this can be done, something like:

 Class naming is done in upper camel case.

```suggestion
 * Use the `class` keyword to define a class.
 * Class naming is done in upper camel case.
 *
 * Create a class named Basics5
```